
from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_Dialog_Graph(object):
    def setupUi(self, Dialog_Graph):
        Dialog_Graph.setObjectName("Dialog_Graph")
        Dialog_Graph.setEnabled(True)
        Dialog_Graph.resize(600, 470)
        Dialog_Graph.setMinimumSize(QtCore.QSize(600, 470))
        Dialog_Graph.setMaximumSize(QtCore.QSize(600, 470))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("UI\\../Images/graph.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        Dialog_Graph.setWindowIcon(icon)
        Dialog_Graph.setLocale(QtCore.QLocale(QtCore.QLocale.Language.Russian, QtCore.QLocale.Country.Kazakhstan))
        self.comboBox_period = QtWidgets.QComboBox(Dialog_Graph)
        self.comboBox_period.setGeometry(QtCore.QRect(10, 60, 111, 21))
        self.comboBox_period.setObjectName("comboBox_period")
        self.comboBox_period.addItem("")
        self.comboBox_period.addItem("")
        self.comboBox_period.addItem("")
        self.label_Periods = QtWidgets.QLabel(Dialog_Graph)
        self.label_Periods.setGeometry(QtCore.QRect(40, 20, 51, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Periods.setFont(font)
        self.label_Periods.setObjectName("label_Periods")
        self.lineEdit_personalized_period = QtWidgets.QLineEdit(Dialog_Graph)
        self.lineEdit_personalized_period.setEnabled(False)
        self.lineEdit_personalized_period.setGeometry(QtCore.QRect(10, 90, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.lineEdit_personalized_period.setFont(font)
        self.lineEdit_personalized_period.setInputMask("")
        self.lineEdit_personalized_period.setText("")
        self.lineEdit_personalized_period.setMaxLength(32767)
        self.lineEdit_personalized_period.setFrame(True)
        self.lineEdit_personalized_period.setEchoMode(QtWidgets.QLineEdit.EchoMode.Normal)
        self.lineEdit_personalized_period.setObjectName("lineEdit_personalized_period")
        self.line = QtWidgets.QFrame(Dialog_Graph)
        self.line.setGeometry(QtCore.QRect(120, 0, 16, 471))
        self.line.setFrameShape(QtWidgets.QFrame.Shape.VLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line.setObjectName("line")
        self.line_2 = QtWidgets.QFrame(Dialog_Graph)
        self.line_2.setGeometry(QtCore.QRect(0, 110, 128, 20))
        self.line_2.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_2.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_2.setObjectName("line_2")
        self.label_Companies = QtWidgets.QLabel(Dialog_Graph)
        self.label_Companies.setGeometry(QtCore.QRect(25, 130, 81, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Companies.setFont(font)
        self.label_Companies.setObjectName("label_Companies")
        self.checkBox_Company_1 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_1.setGeometry(QtCore.QRect(20, 170, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_1.setFont(font)
        self.checkBox_Company_1.setChecked(True)
        self.checkBox_Company_1.setObjectName("checkBox_Company_1")
        self.checkBox_Company_2 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_2.setGeometry(QtCore.QRect(20, 200, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_2.setFont(font)
        self.checkBox_Company_2.setChecked(True)
        self.checkBox_Company_2.setObjectName("checkBox_Company_2")
        self.checkBox_Company_3 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_3.setGeometry(QtCore.QRect(20, 230, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_3.setFont(font)
        self.checkBox_Company_3.setChecked(True)
        self.checkBox_Company_3.setObjectName("checkBox_Company_3")
        self.checkBox_Company_4 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_4.setGeometry(QtCore.QRect(20, 260, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_4.setFont(font)
        self.checkBox_Company_4.setChecked(True)
        self.checkBox_Company_4.setObjectName("checkBox_Company_4")
        self.checkBox_Company_5 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_5.setGeometry(QtCore.QRect(20, 290, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_5.setFont(font)
        self.checkBox_Company_5.setChecked(True)
        self.checkBox_Company_5.setObjectName("checkBox_Company_5")
        self.checkBox_Company_6 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_6.setGeometry(QtCore.QRect(20, 320, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_6.setFont(font)
        self.checkBox_Company_6.setChecked(True)
        self.checkBox_Company_6.setObjectName("checkBox_Company_6")
        self.checkBox_Company_7 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_7.setGeometry(QtCore.QRect(20, 350, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_7.setFont(font)
        self.checkBox_Company_7.setCheckable(True)
        self.checkBox_Company_7.setChecked(True)
        self.checkBox_Company_7.setObjectName("checkBox_Company_7")
        self.checkBox_Company_8 = QtWidgets.QCheckBox(Dialog_Graph)
        self.checkBox_Company_8.setGeometry(QtCore.QRect(20, 380, 91, 20))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.checkBox_Company_8.setFont(font)
        self.checkBox_Company_8.setChecked(True)
        self.checkBox_Company_8.setObjectName("checkBox_Company_8")
        self.label_Parameters = QtWidgets.QLabel(Dialog_Graph)
        self.label_Parameters.setGeometry(QtCore.QRect(318, 10, 91, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(11)
        self.label_Parameters.setFont(font)
        self.label_Parameters.setObjectName("label_Parameters")
        self.line_3 = QtWidgets.QFrame(Dialog_Graph)
        self.line_3.setGeometry(QtCore.QRect(128, 30, 472, 16))
        self.line_3.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_3.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_3.setObjectName("line_3")
        self.line_4 = QtWidgets.QFrame(Dialog_Graph)
        self.line_4.setGeometry(QtCore.QRect(351, 37, 20, 363))
        self.line_4.setFrameShape(QtWidgets.QFrame.Shape.VLine)
        self.line_4.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_4.setObjectName("line_4")
        self.label_Income_Statement = QtWidgets.QLabel(Dialog_Graph)
        self.label_Income_Statement.setGeometry(QtCore.QRect(180, 40, 131, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Income_Statement.setFont(font)
        self.label_Income_Statement.setObjectName("label_Income_Statement")
        self.line_5 = QtWidgets.QFrame(Dialog_Graph)
        self.line_5.setGeometry(QtCore.QRect(128, 240, 232, 16))
        self.line_5.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_5.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_5.setObjectName("line_5")
        self.label_Periods_Decisions = QtWidgets.QLabel(Dialog_Graph)
        self.label_Periods_Decisions.setGeometry(QtCore.QRect(442, 40, 71, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Periods_Decisions.setFont(font)
        self.label_Periods_Decisions.setObjectName("label_Periods_Decisions")
        self.line_6 = QtWidgets.QFrame(Dialog_Graph)
        self.line_6.setGeometry(QtCore.QRect(361, 160, 240, 16))
        self.line_6.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_6.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_6.setObjectName("line_6")
        self.label_Periods_Production = QtWidgets.QLabel(Dialog_Graph)
        self.label_Periods_Production.setGeometry(QtCore.QRect(440, 170, 81, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Periods_Production.setFont(font)
        self.label_Periods_Production.setObjectName("label_Periods_Production")
        self.radioButton_Sales = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Sales.setGeometry(QtCore.QRect(150, 70, 61, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Sales.setFont(font)
        self.radioButton_Sales.setObjectName("radioButton_Sales")
        self.radioButton_COGS = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_COGS.setGeometry(QtCore.QRect(150, 90, 161, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_COGS.setFont(font)
        self.radioButton_COGS.setObjectName("radioButton_COGS")
        self.radioButton_Gross_Margin = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Gross_Margin.setGeometry(QtCore.QRect(150, 110, 111, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Gross_Margin.setFont(font)
        self.radioButton_Gross_Margin.setObjectName("radioButton_Gross_Margin")
        self.radioButton_Layoff_Charge = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Layoff_Charge.setEnabled(False)
        self.radioButton_Layoff_Charge.setGeometry(QtCore.QRect(150, 130, 111, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Layoff_Charge.setFont(font)
        self.radioButton_Layoff_Charge.setObjectName("radioButton_Layoff_Charge")
        self.radioButton_Inventory_Charge = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Inventory_Charge.setGeometry(QtCore.QRect(150, 150, 141, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Inventory_Charge.setFont(font)
        self.radioButton_Inventory_Charge.setObjectName("radioButton_Inventory_Charge")
        self.radioButton_Interest = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Interest.setGeometry(QtCore.QRect(150, 170, 82, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Interest.setFont(font)
        self.radioButton_Interest.setObjectName("radioButton_Interest")
        self.radioButton_Profit_Before_Tax = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Profit_Before_Tax.setGeometry(QtCore.QRect(150, 190, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Profit_Before_Tax.setFont(font)
        self.radioButton_Profit_Before_Tax.setObjectName("radioButton_Profit_Before_Tax")
        self.radioButton_Tax = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Tax.setGeometry(QtCore.QRect(150, 210, 82, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Tax.setFont(font)
        self.radioButton_Tax.setObjectName("radioButton_Tax")
        self.radioButton_Net_Profit = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Net_Profit.setGeometry(QtCore.QRect(150, 230, 81, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Net_Profit.setFont(font)
        self.radioButton_Net_Profit.setObjectName("radioButton_Net_Profit")
        self.radioButton_Price = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Price.setGeometry(QtCore.QRect(380, 70, 82, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Price.setFont(font)
        self.radioButton_Price.setObjectName("radioButton_Price")
        self.radioButton_Marketing = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Marketing.setGeometry(QtCore.QRect(380, 110, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Marketing.setFont(font)
        self.radioButton_Marketing.setObjectName("radioButton_Marketing")
        self.radioButton_Production = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Production.setGeometry(QtCore.QRect(380, 90, 91, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Production.setFont(font)
        self.radioButton_Production.setObjectName("radioButton_Production")
        self.radioButton_Investment = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Investment.setGeometry(QtCore.QRect(380, 130, 101, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Investment.setFont(font)
        self.radioButton_Investment.setObjectName("radioButton_Investment")
        self.radioButton_RD = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_RD.setGeometry(QtCore.QRect(380, 150, 71, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_RD.setFont(font)
        self.radioButton_RD.setObjectName("radioButton_RD")
        self.radioButton_Production_CostUnit = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Production_CostUnit.setGeometry(QtCore.QRect(380, 240, 161, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Production_CostUnit.setFont(font)
        self.radioButton_Production_CostUnit.setObjectName("radioButton_Production_CostUnit")
        self.radioButton_Capacity_Utilization = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Capacity_Utilization.setGeometry(QtCore.QRect(380, 220, 151, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Capacity_Utilization.setFont(font)
        self.radioButton_Capacity_Utilization.setObjectName("radioButton_Capacity_Utilization")
        self.radioButton_Inventory_units = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Inventory_units.setGeometry(QtCore.QRect(380, 260, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Inventory_units.setFont(font)
        self.radioButton_Inventory_units.setObjectName("radioButton_Inventory_units")
        self.radioButton_Factory_Capacity = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Factory_Capacity.setGeometry(QtCore.QRect(380, 200, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Factory_Capacity.setFont(font)
        self.radioButton_Factory_Capacity.setObjectName("radioButton_Factory_Capacity")
        self.radioButton_Employess = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Employess.setEnabled(False)
        self.radioButton_Employess.setGeometry(QtCore.QRect(380, 280, 101, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Employess.setFont(font)
        self.radioButton_Employess.setObjectName("radioButton_Employess")
        self.label_Periods_Balance_Sheet = QtWidgets.QLabel(Dialog_Graph)
        self.label_Periods_Balance_Sheet.setGeometry(QtCore.QRect(193, 250, 101, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Periods_Balance_Sheet.setFont(font)
        self.label_Periods_Balance_Sheet.setObjectName("label_Periods_Balance_Sheet")
        self.radioButton_Cash = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Cash.setGeometry(QtCore.QRect(150, 280, 82, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Cash.setFont(font)
        self.radioButton_Cash.setObjectName("radioButton_Cash")
        self.radioButton_Total_Assets = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Total_Assets.setGeometry(QtCore.QRect(150, 320, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Total_Assets.setFont(font)
        self.radioButton_Total_Assets.setObjectName("radioButton_Total_Assets")
        self.radioButton_Inventory_cost = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Inventory_cost.setGeometry(QtCore.QRect(150, 300, 101, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Inventory_cost.setFont(font)
        self.radioButton_Inventory_cost.setObjectName("radioButton_Inventory_cost")
        self.radioButton_Loans = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Loans.setGeometry(QtCore.QRect(150, 340, 82, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Loans.setFont(font)
        self.radioButton_Loans.setObjectName("radioButton_Loans")
        self.radioButton_Capital = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Capital.setGeometry(QtCore.QRect(150, 360, 81, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Capital.setFont(font)
        self.radioButton_Capital.setObjectName("radioButton_Capital")
        self.radioButton_Liabilities_plus_Equity = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Liabilities_plus_Equity.setGeometry(QtCore.QRect(150, 380, 141, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Liabilities_plus_Equity.setFont(font)
        self.radioButton_Liabilities_plus_Equity.setObjectName("radioButton_Liabilities_plus_Equity")
        self.line_7 = QtWidgets.QFrame(Dialog_Graph)
        self.line_7.setGeometry(QtCore.QRect(361, 290, 240, 16))
        self.line_7.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_7.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_7.setObjectName("line_7")
        self.label_Periods_Marketing = QtWidgets.QLabel(Dialog_Graph)
        self.label_Periods_Marketing.setGeometry(QtCore.QRect(442, 300, 71, 21))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.label_Periods_Marketing.setFont(font)
        self.label_Periods_Marketing.setObjectName("label_Periods_Marketing")
        self.radioButton_Sales_Made = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Sales_Made.setGeometry(QtCore.QRect(380, 320, 161, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Sales_Made.setFont(font)
        self.radioButton_Sales_Made.setObjectName("radioButton_Sales_Made")
        self.radioButton_Unfilled_Orders = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Unfilled_Orders.setGeometry(QtCore.QRect(380, 340, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Unfilled_Orders.setFont(font)
        self.radioButton_Unfilled_Orders.setObjectName("radioButton_Unfilled_Orders")
        self.radioButton_Total_CostUnit_Sold = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_Total_CostUnit_Sold.setGeometry(QtCore.QRect(380, 360, 151, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_Total_CostUnit_Sold.setFont(font)
        self.radioButton_Total_CostUnit_Sold.setObjectName("radioButton_Total_CostUnit_Sold")
        self.radioButton_MarginUnit_Sold = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_MarginUnit_Sold.setGeometry(QtCore.QRect(380, 380, 131, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.radioButton_MarginUnit_Sold.setFont(font)
        self.radioButton_MarginUnit_Sold.setObjectName("radioButton_MarginUnit_Sold")
        self.line_8 = QtWidgets.QFrame(Dialog_Graph)
        self.line_8.setGeometry(QtCore.QRect(0, 392, 601, 16))
        self.line_8.setFrameShape(QtWidgets.QFrame.Shape.HLine)
        self.line_8.setFrameShadow(QtWidgets.QFrame.Shadow.Sunken)
        self.line_8.setObjectName("line_8")
        self.radioButton_MPI = QtWidgets.QRadioButton(Dialog_Graph)
        self.radioButton_MPI.setGeometry(QtCore.QRect(35, 425, 61, 17))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(12)
        self.radioButton_MPI.setFont(font)
        self.radioButton_MPI.setChecked(True)
        self.radioButton_MPI.setObjectName("radioButton_MPI")
        self.pushButton_Cancel = QtWidgets.QPushButton(Dialog_Graph)
        self.pushButton_Cancel.setGeometry(QtCore.QRect(370, 420, 111, 31))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.pushButton_Cancel.setFont(font)
        self.pushButton_Cancel.setObjectName("pushButton_Cancel")
        self.pushButton_Show_Graph = QtWidgets.QPushButton(Dialog_Graph)
        self.pushButton_Show_Graph.setGeometry(QtCore.QRect(240, 420, 111, 31))
        font = QtGui.QFont()
        font.setFamily("Montserrat Medium")
        font.setPointSize(10)
        self.pushButton_Show_Graph.setFont(font)
        self.pushButton_Show_Graph.setObjectName("pushButton_Show_Graph")
        
        self.pushButton_Show_Graph.clicked.connect(self.Show_Graph_was_clicked)
        self.pushButton_Cancel.clicked.connect(Dialog_Graph.close)


        self.retranslateUi(Dialog_Graph)
        
        
        QtCore.QMetaObject.connectSlotsByName(Dialog_Graph)

    def retranslateUi(self, Dialog_Graph):
        _translate = QtCore.QCoreApplication.translate
        Dialog_Graph.setWindowTitle(_translate("Dialog_Graph", "Graphing data"))
        self.comboBox_period.setItemText(0, _translate("Dialog_Graph", "0 - current "))
        self.comboBox_period.setItemText(1, _translate("Dialog_Graph", "current-2 - current"))
        self.comboBox_period.setItemText(2, _translate("Dialog_Graph", "Personalized"))
        self.label_Periods.setText(_translate("Dialog_Graph", "Periods"))
        self.lineEdit_personalized_period.setPlaceholderText(_translate("Dialog_Graph", "For example: 1-10, 8, 11-13"))
        self.label_Companies.setText(_translate("Dialog_Graph", "Companies"))
        self.checkBox_Company_1.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_2.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_3.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_4.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_5.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_6.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_7.setText(_translate("Dialog_Graph", "CheckBox"))
        self.checkBox_Company_8.setText(_translate("Dialog_Graph", "CheckBox"))
        self.label_Parameters.setText(_translate("Dialog_Graph", "Parameters"))
        self.label_Income_Statement.setText(_translate("Dialog_Graph", "Income Statement"))
        self.label_Periods_Decisions.setText(_translate("Dialog_Graph", "Decisions"))
        self.label_Periods_Production.setText(_translate("Dialog_Graph", "Production"))
        self.radioButton_Sales.setText(_translate("Dialog_Graph", "Sales"))
        self.radioButton_COGS.setText(_translate("Dialog_Graph", "COGS (Cost of Goods)"))
        self.radioButton_Gross_Margin.setText(_translate("Dialog_Graph", "Gross Margin"))
        self.radioButton_Layoff_Charge.setText(_translate("Dialog_Graph", "Layoff Charge"))
        self.radioButton_Inventory_Charge.setText(_translate("Dialog_Graph", "Inventory Charge"))
        self.radioButton_Interest.setText(_translate("Dialog_Graph", "Interest"))
        self.radioButton_Profit_Before_Tax.setText(_translate("Dialog_Graph", "Profit before Tax"))
        self.radioButton_Tax.setText(_translate("Dialog_Graph", "Tax"))
        self.radioButton_Net_Profit.setText(_translate("Dialog_Graph", "Net Profit"))
        self.radioButton_Price.setText(_translate("Dialog_Graph", "Price"))
        self.radioButton_Marketing.setText(_translate("Dialog_Graph", "Marketing"))
        self.radioButton_Production.setText(_translate("Dialog_Graph", "Production"))
        self.radioButton_Investment.setText(_translate("Dialog_Graph", "Investment"))
        self.radioButton_RD.setText(_translate("Dialog_Graph", "R and D"))
        self.radioButton_Production_CostUnit.setText(_translate("Dialog_Graph", "Production Cost/Unit"))
        self.radioButton_Capacity_Utilization.setText(_translate("Dialog_Graph", "Capacity Utilization"))
        self.radioButton_Inventory_units.setText(_translate("Dialog_Graph", "Inventory (units)"))
        self.radioButton_Factory_Capacity.setText(_translate("Dialog_Graph", "Factory Capacity"))
        self.radioButton_Employess.setText(_translate("Dialog_Graph", "Employees"))
        self.label_Periods_Balance_Sheet.setText(_translate("Dialog_Graph", "Balance Sheet"))
        self.radioButton_Cash.setText(_translate("Dialog_Graph", "Cash"))
        self.radioButton_Total_Assets.setText(_translate("Dialog_Graph", "Total Assets"))
        self.radioButton_Inventory_cost.setText(_translate("Dialog_Graph", "Inventory ($)"))
        self.radioButton_Loans.setText(_translate("Dialog_Graph", "Loans"))
        self.radioButton_Capital.setText(_translate("Dialog_Graph", "Capital"))
        self.radioButton_Liabilities_plus_Equity.setText(_translate("Dialog_Graph", "Liabilities+Equity"))
        self.label_Periods_Marketing.setText(_translate("Dialog_Graph", "Marketing"))
        self.radioButton_Sales_Made.setText(_translate("Dialog_Graph", "Sales Made"))
        self.radioButton_Unfilled_Orders.setText(_translate("Dialog_Graph", "Unfilled Orders"))
        self.radioButton_Total_CostUnit_Sold.setText(_translate("Dialog_Graph", "Total Cost/Unit Sold"))
        self.radioButton_MarginUnit_Sold.setText(_translate("Dialog_Graph", "Margin/Unit Sold"))
        self.radioButton_MPI.setText(_translate("Dialog_Graph", "MPI"))
        self.pushButton_Cancel.setText(_translate("Dialog_Graph", "Cancel"))
        self.pushButton_Show_Graph.setText(_translate("Dialog_Graph", "Show Graph"))
        
    
    def Show_Graph_was_clicked(self):
        pass  
    
    
if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Dialog_Graph = QtWidgets.QWidget()
    ui = Ui_Dialog_Graph()
    ui.setupUi(Dialog_Graph)
    Dialog_Graph.show()
    sys.exit(app.exec())
